python/py-sure/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-sure
+version             1.4.11
+revision            0
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-3+
+maintainers         nomaintainer
+
+description         Utility belt for automated testing in python for python
+long_description    ${description}
+
+homepage            http://github.com/gabrielfalcao/sure
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  3c8d5271fb18e2c69e2613af1ad400d8df090f1456081635bd3171847303cdaa \
+                    rmd160  fb6c37da7c2d005c6874d266fbc8d2edcb21d833 \
+                    size    45933
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    post-extract {
+        system "find ${worksrcpath} -type d -print0 | xargs -0 chmod a+rx"
+        system "find ${worksrcpath} -type f -print0 | xargs -0 chmod a+r"
+    }
+
```

Comment:
here you'll need:

```
depends_build-append \
                    port:py${python.version}-nose
```

Reply:
It seems to me that trace-mode (or something else) isn't working correctly.... With the current PR when I do ```port -dvst test py37-sure``` it actually downloads the missing dependencies from PyPI, which in my understanding should not happen....

Relevant part from the log for the first package:
```
Executing:  cd "/opt/local/var/macports/build/_Users_renee_Software_macports-ports_python_py-sure/py37-sure/work/sure-1.4.11" && /opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 setup.py --no-user-cfg test
DEBUG: system:  cd "/opt/local/var/macports/build/_Users_renee_Software_macports-ports_python_py-sure/py37-sure/work/sure-1.4.11" && /opt/local/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 setup.py --no-user-cfg test
running test
Searching for six
Reading https://pypi.org/simple/six/
Downloading https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl#sha256=3350809f0555b11f552448330d0b52d5f24c91a322ea4a15ef22629740f3761c
Best match: six 1.12.0
Processing six-1.12.0-py2.py3-none-any.whl
Installing six-1.12.0-py2.py3-none-any.whl to /opt/local/var/macports/build/_Users_renee_Software_macports-ports_python_py-sure/py37-sure/work/sure-1.4.11/.eggs
```

Reply:
(That's why I deleted the comment before you wrote this reply.)